### DISCUSSION TITLE
How to cache `node_modules`

### DISCUSSION THREAD

**Discussion by italodeandra** [caprover/caprover#1280] [Q&A] [answered]
Hi team,

My `yarn install` on the build process is taking more than 6 minutes to install every time I deploy the app. This is too long for a simple Next.js application with very few dependencies. I think that's probably because `node_modules` it not cached. Is there anything I could do to cache it?

**Comment by githubsaturn** [comment, marked as answer]
Generally speaking, Docker does that automatically for you unless something has changed.

Regardless, CapRover does not modify this at all, this is mostly a Docker question, see here for an example: 
https://stackoverflow.com/questions/35616170/how-to-cache-node-modules-on-docker-build
